easybuild/tools/modules.py

```diff
@@ -1014,6 +1038,7 @@ class EnvironmentModulesTcl(EnvironmentModulesC):
     COMMAND_SHELL = ['tclsh']
     VERSION_OPTION = ''
     REQ_VERSION = None
+    MAX_VERSION = '1.923'
```

Comment:
I would leave this unspecified, there's little point to it?

If somebody uses `EnvironmentModulesTcl` with Modules v4.0, it will fail anyway because `eb` can't determine the version?